runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -637,9 +542,7 @@ public void completeExceptionally(long requestID, Throwable cause) {
      * @param ctx The context of the channel handler.
      * @return True, if the epoch is correct, but false otherwise.
      */
-    @Deprecated // TODO: Add replacement method that conforms to style
-    @SuppressWarnings("checkstyle:abbreviation") // Due to deprecation
-    private boolean validateEpochAndClientID(CorfuMsg msg, ChannelHandlerContext ctx) {
+    private boolean validateEpoch(CorfuMsg msg, ChannelHandlerContext ctx) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "ctx". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)